spec/index.html

```diff
@@ -3167,29 +3167,44 @@ <h5>The <code>&lt;<dfn id="the-coordinates-element">coordinates</dfn>&gt;</code>
         </p>
         </section>
         </section>
+        
+        <section class="informative">
+        <h3>Authoring</h3>
+        <p>
+        While MapML is human-readable and can be written by hand,
+        authors typically will use conversion programs
+        and other specialized software tools to generate MapML.
+        </p>
+        
+        <section>
+        <h4 id="examples">Examples</h4>
+          <aside class="example">
+            <figure>
+              <a href="examples/mapml.png" style="cursor:zoom-in;">
+                <img src="examples/mapml.png" width="3240" height="2016" loading="lazy" alt="Origins, orientations and axis names of tcrs, pcrs, gcrs, tilematrix, map, and tile coordinate systems">
+              </a>
+              <figcaption>
+              Prototype MapML services are available,
+              representing
+              <a href="https://geogratis.gc.ca/mapml/en/osmtile/cbmt/">tiled</a>
+              and
+              <a href="https://geogratis.gc.ca/api/beta/vectors/canvec/50k/features/?entry-type=full&xmin=-75.6088650226593&ymin=45.39715420088217&xmax=-75.59665560722351&ymax=45.401975540403974&zoom=17&projection=WGS84&alt=xml">
```

Comment:
This links (https://geogratis.gc.ca/api/beta/vectors/canvec/50k/features/?entry-type=full&xmin=-75.6088650226593&ymin=45.39715420088217&xmax=-75.59665560722351&ymax=45.401975540403974&zoom=17&projection=WGS84&alt=xml) does not work atm. @prushforth should it be replaced with another URL?

Reply:
Eventually we'll have a replacement, but that link is likely gone forever. Perhaps change the sentence for now.

Reply:
I removed the link. Please suggest something if you want the sentence changed, otherwise I think we can merge this.